spec/intl.html

```diff
@@ -1685,20 +1685,72 @@ <h1>
           </dl>
         </emu-clause>
 
-        <emu-clause id="sec-temporal-calendardatemergefields" type="abstract operation">
+        <emu-clause id="sec-temporal-calendarfieldkeystooverride" type="abstract operation">
           <h1>
-            CalendarDateMergeFields (
+            CalendarFieldKeysToOverride (
```

Comment:
Minor bikeshedding thing: "override" doesn't necessarily seem like the right word to describe what's happening, because `addtionalFields` often doesn't contain the fields being "overridden". The goal of this AO is to figure out which fields should be removed from `fields` so that the result, when merged, won't be ambiguous. So maybe some other possible names could be one of the following, or some other clearer name?

* FieldKeysToRemove
* FieldKeysToIgnore
* AmbiguousFieldKeys
* ConflictingFieldKeys (or FieldKeysThatConflict)

Reply:
Any of those are fine, but I'd like move towards ending the bikeshedding, and pick one that everybody can live with and stick with it. I've changed the name a few times already, and had to rephrase the accompanying prose each time.

Reply:
Deal. I'd suggest `*FieldKeysToIgnore` but any of those are OK with me.

Reply:
There were no other comments, so I'll move ahead with that. If we need to iterate further on the name or the phrasing, that can be done in a separate PR.